xml/ntp_banners.xml

```diff
@@ -419,49 +419,247 @@ NTP "banners", taken from a readvar response
     <param pos="0" name="os.family" value="NetWare"/>
     <param pos="0" name="os.product" value="NetWare"/>
   </fingerprint>
-  <fingerprint pattern="^.*version=&quot;ntpd ([^ ]+)[^&quot;]+&quot;,.*processor=&quot;([^ ]+)&quot;,.*system=&quot;SunOS/?([^ ]+)&quot;" flags="REG_DOT_NEWLINE,REG_ICASE">
-    <description>ntpd running on Solaris</description>
+  <fingerprint pattern="^.*version=&quot;ntpd ([^ ]+)[^&quot;]+&quot;,.*processor=&quot;([^ ]+)&quot;,.*system=&quot;SunOS/5.0&quot;" flags="REG_DOT_NEWLINE,REG_ICASE">
+    <description> ntpd running on Solaris 2.0 (SunOS/5.0) </description>
```

Comment:
Will update the regex back to the original format with /? to take account of the strings where it is 'SunOS5.x' as per that shodan.io we just looked at and add example cases, thanks @ncameron-r7 

Link for anyone else interested: https://www.shodan.io/search?query=SunOS5.6